conf/config.json

```diff
@@ -97,12 +105,61 @@
             "monitorReplicationFailureExpiryTimeS": 86400,
             "queueProcessor": {
                 "groupId": "backbeat-replication-group",
-                "retryTimeoutS": 300,
+                "retry": {
+                    "aws": {
+                        "maxRetries": 5,
+                        "timeoutS": 300,
+                        "backoff": {
+                            "min": 1000,
```

Comment:
Didn't you want to take the opportunity to adjust the default values for all clouds, e.g. setting a larger min backoff and larger timeout by default? Not perfect for all environments of course but hopefully closer to reasonable defaults in production env.